packages/wonder-blocks-tabs/src/components/tabs.tsx

```diff
@@ -174,10 +212,34 @@ export const Tabs = React.forwardRef(function Tabs(
     const visitedTabsRef = React.useRef(new Set<string>());
     visitedTabsRef.current.add(selectedTabId);
 
+    /**
+     * Ref for the tablist.
+     */
+    const tablistRef = React.useRef<HTMLDivElement>(null);
+
+    /**
+     * Determine if the tab is active if it has aria-selected="true".
+     */
+    const isTabActive = React.useCallback((tabElement: Element): boolean => {
+        return tabElement.ariaSelected === "true";
+    }, []);
+
+    const {indicatorProps, updateUnderlineStyle} = useTabIndicator({
```

Comment:
This uses the same logic as NavigationTabs for the selected tab indicator animation. 

I created https://khanacademy.atlassian.net/browse/WB-1924 for adding animations to the tab panel. This is something that won't be needed for NavigationTabs since that relies on link navigation.